Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair_degeneracies.h

```diff
@@ -1861,12 +1861,6 @@ bool remove_degenerate_faces(const FaceRange& face_range,
   if(degenerate_face_set.empty())
     return true;
 
-  if(degenerate_face_set.size() == faces_size)
```

Comment:
Thinking about it, I think I would keep that and update the clear function for Surface_mesh to call the version that is keeping properties. as `clear()` is meant to be used by CGAL developers within there algo, it makes senses not to mess up with user properties.